### PR TITLE
Update android-components branch; support ignoring pushes to main like we do for master

### PR DESCRIPTION
**Pull request by bhearsum**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -38,6 +38,7 @@ COPY manage.py newrelic.ini version.json /app/
 COPY --from=frontend /app/.build/ /app/.build/
 
 RUN python manage.py collectstatic --noinput
+RUN python -m whitenoise.compress .build
```
Comment (sarah-clements) [inline docker/Dockerfile:41]: Is there a reason you'd want to add it here, since it's already in #7219 ?

Comment (bhearsum) [reply docker/Dockerfile:41]: no, that's an accident - whoops!

**Review by sarah-clements — APPROVED** [review]
I think this looks ok. Not sure if we also want to ignore branches in the push ingestion or not, instead of just tasks.

**Comment by bhearsum** [comment]
> I think this looks ok. Not sure if we also want to ignore branches in the push ingestion or not, instead of just tasks.

We're not doing it now AFAICT, so it's probably okay?

**Pull request merged**